CHANGELOG.md

```diff
@@ -1,2 +1,21 @@
 # Version 6.0.0
 Released June 14, 2021
+* Added telemetry. The sequelize version is recorded when creating a new instance of a CockroachDB Sequelize instance.
```

Comment:
Could you also include a bullet for "Initial support for Sequelize 6.0"

Reply:
Done